WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -0,0 +1,221 @@
+import UIKit
+
+class CommentContentTableViewCell: UITableViewCell, NibReusable {
+
+    // determines the state of the like button.
+    enum LikeState {
+        case unliked
+        case liked
+    }
+
+    // all the available images to display for the accessory button.
+    enum AccessoryButtonType {
+        case share
+        case threeDots
+    }
+
+    // MARK: - Public Properties
+
+    var displayName: String? = "" {
+        didSet {
+            nameLabel?.setText(displayName ?? "")
+        }
+    }
+
+    var displayDate: String? = "" {
+        didSet {
+            dateLabel?.setText(displayDate ?? "")
+        }
+    }
+
+    var accessoryButtonType: AccessoryButtonType = .share {
+        didSet {
+            accessoryButton.setImage(accessoryButtonImage, for: .normal)
+        }
+    }
+
+    var numberOfLikes: Int = 0 {
+        didSet {
+            updateLikeButton()
+        }
+    }
+
+    var likeButtonState: LikeState = .unliked {
+        didSet {
+            updateLikeButton()
+        }
+    }
+
+    // MARK: Button Tap Handlers
+
+    var nameLabelTapAction: (() -> Void)? = nil
+
+    var accessoryButtonAction: (() -> Void)? = nil
+
+    var replyButtonAction: (() -> Void)? = nil
+
+    var likeButtonAction: (() -> Void)? = nil
+
+    // MARK: Component Visibility
+
+    // Hides the accessory button if the value is false.
+    var accessoryButtonEnabled: Bool = true {
+        didSet {
+            accessoryButton?.isHidden = !accessoryButtonEnabled
+        }
+    }
+
+    var replyButtonEnabled: Bool = true {
+        didSet {
+            replyButton?.isHidden = !replyButtonEnabled
+        }
+    }
+
+    var likeButtonEnabled: Bool = true {
+        didSet {
+            likeButton?.isHidden = !likeButtonEnabled
+        }
+    }
+
+    // the Reply and Like buttons will be hidden if this is set to false.
+    var reactionBarEnabled: Bool = true {
+        didSet {
+            reactionBarView?.isHidden = !reactionBarEnabled
+        }
+    }
+
+    // MARK: Outlets
+
+    @IBOutlet private weak var avatarImageView: CircularImageView!
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var dateLabel: UILabel!
+    @IBOutlet private weak var accessoryButton: UIButton!
+
+    @IBOutlet weak var webView: WKWebView!
+
+    @IBOutlet private weak var reactionBarView: UIView!
+    @IBOutlet private weak var replyButton: UIButton!
+    @IBOutlet private weak var likeButton: UIButton!
+
+    // MARK: Lifecycle
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureViews()
+    }
+
+    // MARK: Public Methods
+
+    /// Configures the avatar image view with the provided URL.
+    /// If the URL does not contain any image, the default placeholder image will be displayed.
+    /// - Parameter url: The URL containing the image.
+    func configureImage(with url: URL?) {
```

Comment:
Again I'm assuming there will be a `configure` type method that will accept the `Comment`, which would call these image configure methods? Meaning they can be private?

Reply:
👍 . Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17109/commits/05d5a31e08a13244e2a493f9cf2aea0b40f07f88.